types/index.ts

```diff
@@ -45,9 +45,9 @@ export type Library = {
     license: {
       key: string;
       name: string;
-      spdx_id: string;
+      spdxId: string;
```

Comment:
These two fields have been updated by graphql. I couldn't see them being used anywhere but it made more sense to just leave them in camelcase to match the rest of the output rather than having to modify them. I can revert this if it's going to cause a breaking change.